packages/vuetify/src/components/VSelect/VSelect.ts

```diff
@@ -349,7 +346,7 @@ export default VTextField.extend({
         key
       }), `${this.getText(item)}${last ? '' : ', '}`)
     },
-    genDefaultSlot () {
+    genDefaultSlot (): any[] {
```

Comment:
`: VNode[]` (or whatever it returns)

Reply:
It can return array of VNodes and VNode arrays